docker-entrypoint.sh

```diff
@@ -80,7 +80,16 @@ if expr "$*" : '^java .*/start\.jar.*$' >/dev/null ; then
 		set -- $(cat /jetty-start)
 	else
 		# Do a jetty dry run to set the final command
-		set -- $("$@" --dry-run | sed 's/\\$//' )
+		"$@" --dry-run > /jetty-start
+		EXIT=$?
```

Comment:
Good point.   But if it exits, we lose any stdout that might explain why... although I guess it should put any errors onto stderr.

Two choices:
a) don't capture the exit code and just exit 0 if there are more than 1 lines in the output.
b) turn off -e

I'm good with a) because looking at start.jar code, it appears to write to stderr before any non zero exit